src/cljs/lomake_editori/editor/component.cljs

```diff
@@ -39,14 +42,17 @@
                 (for [lang @languages]
                   ^{:key lang}
                   [:input.editor-form__text-field {:value     (get-in @value [:label lang])
-                                       :on-change #(dispatch [:editor/set-component-value (-> % .-target .-value) path :label lang])}]))]])
+                                                   :on-change #(dispatch [:editor/set-component-value (-> % .-target .-value) path :label lang])}]))]])
           (into
             [[:div.editor-form__size-button-wrapper
               [:header.editor-form__component-item-header "Koko"]
-              [:div.editor-form__size-button-group
-               [:span.editor-form__size-button.editor-form__size-button--not-selected "S"]
-               [:span.editor-form__size-button.editor-form__size-button--selected "M"]
-               [:span.editor-form__size-button.editor-form__size-button--not-selected "L"]]]])
+              [:div.editor-form__size-button-group {:on-change #(do)}
+               (map #(seq (let [btn-name (key %)
+                                btn-id (val %)]
```

Comment:
maybe btn-key instead? I thought that there's an actual (evil :) ) html id involved first :)